src/private/privateAPIs.ts

```diff
@@ -12,6 +12,10 @@ import { getGenericOnCompleteHandler } from '../internal/utils';
 import { Communication, sendMessageToParent, sendMessageEventToChild } from '../internal/communication';
 import { menus } from './menus';
 import { registerHandler } from '../internal/handlers';
+import { GlobalVars } from '../internal/globalVars';
+import { ErrorCode, SdkError } from '../public/interfaces';
+
+const getUserJoinedTeamsSupportedAndroidClientVersion = '2.0.1';
```

Comment:
Could you please move all min required versions to internal const file. Wanted to keep all hardcoded version at one place.

Reply:
Have moved the hardcoded versions to the internal constants file. Please take a look.

Reply:
@sachingupta could you please review the changes.